### ISSUE TITLE
-Wunneeded-internal-declaration in drivers/cpufreq/cppc_cpufreq.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#178]
```bash
drivers/cpufreq/cppc_cpufreq.c:431:36: warning: variable 'cppc_acpi_ids' is not needed and will not be emitted [-Wunneeded-internal-declaration]
static const struct acpi_device_id cppc_acpi_ids[] = {
                                   ^
1 warning generated.
```

Should probably be fixed in the same manner as #169. Once that solution has been given the thumbs up by upstream, I'll send a patch for this one.

**Comment by nickdesaulniers** [comment]
> Should probably be fixed in the same manner as #169.

I agree.

**Comment by nathanchance** [comment]
Pending patch: https://github.com/nathanchance/linux/commit/562d6f49cbf51727a8a9c03ba94668df346344f2

**Comment by nathanchance** [comment]
Patch sent: https://lore.kernel.org/lkml/20181002223457.7564-1-natechancellor@gmail.com/

**Comment by nathanchance** [comment]
Patch accepted: https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=8ff3c22688ff3616ee53c5c54b3ee66a73326175

**Comment by nathanchance** [comment]
Merged into mainline: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=8ff3c22688ff3616ee53c5c54b3ee66a73326175

**Comment by nickdesaulniers** [comment]
possibly also fixed on the llvm side by #232
